examples/install_requirements.py

```diff
@@ -114,3 +114,23 @@ def hasWhitespace(string):
     prettyPrint(downloader_cmd)
 else:
     subprocess.check_call(downloader_cmd)
+
+# Use when updating blobs to new OpenVINO version
```

Comment:
Let's add a CLI option and default it to model downloader

Reply:
@szabi-luxonis Added `--convert` option.
If provided as `--convert <version>`, it will convert the models using the specified OpenVINO version. If only `--convert` was provided, it will use the default version.